integration-tests/utils/integration-utils.js

```diff
@@ -169,10 +169,15 @@ const dismissNotifications = async page => {
   return !!notificationCloseButtons.length && delay(1000) // delayed for alerts to animate off
 }
 
-const signIntoTerra = async (page, token) => {
-  await page.waitForXPath('//*[contains(normalize-space(.),"Loading Terra")]', { hidden: true })
+const signIntoTerra = async (page, { token, testUrl }) => {
+  if (testUrl) {
+    await page.goto(testUrl, waitUntilLoadedOrTimeout(60 * 1000))
+  }
```

Comment:
We typically one-line simple conditionals, like so:

```suggestion
  !!testUrl && await page.goto(testUrl, waitUntilLoadedOrTimeout(60 * 1000))
```

`!!` is to be explicit about our intention of coercing a non-boolean (`testUrl` in this case) into a truthy/falsy.

Reply:
Thanks.